internal/cloud/realm/dependencies_test.go

```diff
@@ -28,8 +28,32 @@ func TestRealmDependencies(t *testing.T) {
 		assert.Nil(t, err)
 
 		uploadPath := filepath.Join(wd, "testdata/dependencies_upload.zip")
+
+		_, err = client.DependenciesStatus(groupID, app.ID)
+		assert.Equal(t, realm.ServerError{Message: "dependency installation not found"}, err)
+
 		assert.Nil(t, client.ImportDependencies(groupID, app.ID, uploadPath))
 
+		startStatus, err := client.DependenciesStatus(groupID, app.ID)
+		assert.Nil(t, err)
+		assert.Equal(t, realm.DependenciesStatus{
+			State:   realm.DependenciesStateCreated,
+			Message: "processing request",
+		}, startStatus)
+
+		// wait for dependencies to finish installation
+		var currentStatus realm.DependenciesStatus
+		for i := 0; i < 50; i++ {
```

Comment:
in my computer it usually takes around <15 runs but I imagine with all sorts of delays and all, `50` seems to be a good balance between making sure it isn't flaky but also not running for too long; could be lowered.